lib/atrium/institution.rb

```diff
@@ -1,17 +1,37 @@
+require "URI"
+
 module Atrium
   class Institution
-    ##
-    # GET /institutions/:institution_code/credentials
-    #
+    include ::ActiveAttr::Model
+
+    attribute :code
+    attribute :name
+    attribute :url
+
     def self.credentials(institution_code)
-      ::Atrium.client.make_request(:get, "/institutions/#{institution_code}/credentials")
+      endpoint = "/institutions/#{institution_code}/credentials"
+      response = ::Atrium.client.make_request(:get, endpoint)
+
+      response["credentials"].map do |credential|
+        ::Atrium::Credential.new(credential)
+      end
     end
-    
-    ##
-    # GET /institutions
-    #
-    def self.list
-      ::Atrium.client.make_request(:get, "/institutions")
+
+    def self.list(options = nil)
+      endpoint = format_endpoint("institutions", options)
+      response = ::Atrium.client.make_request(:get, endpoint)
```

Comment:
So this list will only get the first page? Do we need to check if there are additional pages we need to fetch based on the response?